react/components/AutoComplete/AutoComplete.stories.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import AutoComplete from "./AutoComplete";
+
+export default {
+    title: "AutoComplete",
+    component: AutoComplete,
+};
+
+const options = ["Option1", "Option2", "Option3", "Option4", "Option5"];
```

Comment:
Do we find it likely that this will be the shape of our options? 
My concern is our options may be objects that are required to send to the backend. This may be lost here.

Reply:
ah nevermind
I see AA can take values like 
const options = [
  { label: 'The Godfather', id: 1 },
  { label: 'Pulp Fiction', id: 2 },
];